bucket/desktopnote.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "b59f84e788b07ee6b3d619b509b9f709376f2135",
```

Comment:
This is really awefull.

Reply:
Maybe b59f84e would be better?

Reply:
I would rather wait for proper release cycle rather than commits.

Reply:
There is no release for more than 3 years and I don't think ther will be :joy:

Reply:
Latest commit to master_C# was 2 hours back.

Reply:
Yeah, I ask for a license and a MIT license was added.